apps/files/models.py

```diff
@@ -581,6 +581,8 @@ class FileUpload(amo.models.ModelBase):
         choices=amo.APPS_CHOICES, db_column="compat_with_app_id", null=True)
     compat_with_appver = models.ForeignKey(
         AppVersion, null=True, related_name='uploads_compat_for_appver')
+    version = models.CharField(max_length=255, null=True)
+    addon = models.ForeignKey('addons.Addon', null=True)
```

Comment:
A `File` belongs to a `Version` so it isn't created until the upload passes validation and we [create a `Version`](https://github.com/mozilla/olympia/blob/02acf2283c2cb218382f0d00ad8340fb28083678/apps/versions/models.py#L156-L158).

Currently every `Version` that isn't reviewed goes in the reviewer queue. We could hypothetically create some new statuses like `amo.STATUS_VALIDATING` and `amo.STATUS_VALIDATION_FAILED` to hide from the queue and create the `Version` right away but that seems like a pretty big undertaking.